### ISSUE TITLE
Verifier's Dilemma example is broken

### ISSUE THREAD

**Issue by ryan10** [BlockScience/cadCAD-Tutorials#23]
Verifier's dilemma example references SimCAD package - is that a former name for the cadCAD package?

**Comment by mzargham** [comment]
yes. it was in the internal name of cadCAD before we released it. We changed it because there is already something called simcad.

**Comment by mzargham** [comment]
Currently, this repo is out of sync with the current version of cadCAD but we've been holding out an overhaul as there are other standards and best practices coming down the pike. However, if you go through and modify the imports and the calls to the engine API to match the current version of cadCAD we'll happily take the pull request.
